indexer_util/indexer_util/indexer_util.py

```diff
@@ -90,10 +91,11 @@ def init_elasticsearch(elasticsearch_url, index_name):
 
     _wait_elasticsearch_healthy(es)
 
-    logger.info('Deleting and recreating %s index.' % index_name)
-    try:
-        es.indices.delete(index=index_name)
-    except Exception:
-        pass
-    es.indices.create(index=index_name, body={})
+    if es.indices.exists(index=index_name):
+        logger.info(
+            '%s index already exists at %s.' % (index_name, elasticsearch_url))
```

Comment:
nit: the phrasing here makes it seem like an error even though this is a perfectly valid use case. Maybe something like `Using existing index %s at %s`?